CHANGELOG.md

```diff
@@ -13,23 +13,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Include Card Separator example in Card kit docs ([#439][] @rruiz85)
 - Text Input Dark mode added, Text Input updated to use Textarea mixins and Textarea mixin fix to remove number field arrows ([#494][] @megantrimble)
 - Introduce Pull Request Template ([#508][] @rafbgarcia)
+- Extend Form Kit to use Textarea Kit for textarea form fields ([#506][] @roxannecojocariu)
 
 [#498]: https://github.com/powerhome/playbook/pull/498
 [#439]: https://github.com/powerhome/playbook/pull/439
+[#494]: https://github.com/powerhome/playbook/pull/494
```

Comment:
@megantrimble Just wanted to point out that you hadn't added a link to your PR here so I did it for you (just in case there is any confusion).